blueprints/nvidia-gpu-workload/README.md

```diff
@@ -80,16 +68,14 @@ spec:
   subnetSelectorTerms:
   - tags:
       karpenter.sh/discovery: $CLUSTER_NAME
-  tags:
```

Comment:
FYI - these tags on the nodes don't make a lot of sense (to me). The discovery tag is intended to be placed on resources that Karpenter will query for use (i.e. - subnets, security groups, etc.). These are unlikely to be of any use on the nodes deployed